contrib/opencensus-correlation/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.dev0'
+__version__ = '0.4.dev0'
```

Comment:
Dev releases are ordered before corresponding regular releases (see https://www.python.org/dev/peps/pep-0440/#developmental-releases), so this should generally be one minor version ahead of the latest released version.